modules/core/src/main/java/org/apache/ignite/internal/suggestions/ConfigurationSuggestions.java

```diff
@@ -31,10 +32,12 @@
 /**
  * Grid performance suggestions.
  */
-public class GridPerformanceSuggestions {
+public class ConfigurationSuggestions {
     /** */
     private static final boolean disabled = Boolean.getBoolean(IGNITE_PERFORMANCE_SUGGESTIONS_DISABLED);
 
+    private static final String SUGGESTIONS_LINK = "https://apacheignite.readme.io/docs/jvm-and-system-tuning";
```

Comment:
> Also, can we check if OS is CentOS based and check as much OS properties as we can?
Yes, I can do it, but I think will better do it within the separate task where it will be described.

> ..configure gc logging...
Check on existence "-Xloggc"?